### DISCUSSION TITLE
using Shopify Hydrogen, the get started is not working.

### DISCUSSION THREAD

**Discussion by iska31** [sergiodxa/remix-i18next#134] [Q&A] [answered]
Following your get started I'm having issue :
```
 ✘ [ERROR] Could not resolve "fs/promises"

    node_modules/@remix-run/node/dist/upload/fileUploadHandler.js:17:23:
      17 │ var promises = require('fs/promises');
         ╵                        ~~~~~~~~~~~~~

  The package "fs/promises" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.


Build failed with 3 errors:
app/entry.server.tsx:45:22: ERROR: Could not resolve "node:path"
app/i18next.server.ts:2:22: ERROR: Could not resolve "node:path"
node_modules/@remix-run/node/dist/upload/fileUploadHandler.js:17:23: ERROR: Could not resolve "fs/promises"
```

Changing `serverPlatform: 'neutral'` for `node`doesn't help.

Any idea on how to debug that ?

**Comment by sergiodxa** [comment, marked as answer]
I imagine you're using Oxygen, the Oxygen runtime doesn't use Node, it's similar to Cloudflare, so you can't use `fs/promises` or `node:path` because they're Node things.

For your case, use a different backend to load your locales, for example a fetch based one.

**Reply by magnattic** [reply]
I'm stuck on the same problem. Having a working example of how to use this library with hydrogen would really be great!
Did anyone figure out how to do this?

**Reply by magnattic** [reply]
I found a solution by switching out the node backend against an [i18next-fetch-backend](https://www.npmjs.com/package/i18next-fetch-backend) and fetching the asset files from the oxygen CDN. [This PR here](https://github.com/Shopify/hydrogen/pull/1279) has some good code snippets to get you started, specifically the [`getOxygenAssets.server.ts`](https://github.com/Shopify/hydrogen/pull/1279/files#diff-57719255e8035721c5635187f30644de8b94e734bc1ff2d516acfe4ebbdfc899) helped me.

**Comment by Makxzod** [comment]
Any news about this?

**Reply by sergiodxa** [reply]
An answer was provided on another message, you can’t use the fs backend. I never used Hydrogen myself so it would be better to ask the Hydrogen team to create an example for using remix-i18next.

Also note that loading translation files is outside the scope of this package since that’s configured directly for i18next using a backend.

**Comment by ghost** [comment]
Hi guys, anyone has a working xample for **Remix v2** and **Shopify hydrogen**?
